### PR TITLE
SignUp page made

### PR DESCRIPTION
**Pull request by Asypher00**
check signup.css for code
Screenshot (25)

### CODE REVIEW

File: signup.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    flex-wrap: wrap;
+    align-items :center;
+}
+.nav-container{
+    background-color: rgb(188, 251, 62);
+ align-items: center;
+justify-content: space-between;
+width : 97vw;
+
+}
```
Comment (tanishq20) [inline signup.css]: you can use prettier to format it well.

File: signup.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    border : 2px solid black;
+    border-radius: 10px;
+   }
+   
+   .signup-container{
+       width : 20vw;
+       align-items: center;
+       display: flex;
+       flex-direction: column;
+       background-color:rgb(188, 251, 62) ;
```
Comment (tanishq20) [inline signup.css]: var() is preferred for colors.
It is reusable in all files.

**Review by tanishq20 — APPROVED** [review]
Code looks good.
Suggested some minor changes. You can refactor it later on.

**Comment by TJain02** [comment]
Give spacing for cleaner code. Rest looks good.

**Pull request merged**